app/controllers/auth_controller.rb

```diff
@@ -31,6 +31,15 @@ def fetch_return_to_path
   def do_user_auth(auth)
     sign_out
     user = User.from_oauth_hash(auth)
+    unless ENV["NO_WELCOME_EMAIL"]
```

Comment:
why do we have an env var for this? not sure why we'd want to suppress welcome emails but not any others